src/pages/o/[orgId]/campaigns.tsx

```diff
@@ -7,15 +9,15 @@ import getOrg from '../../../fetching/getOrg';
 
 export const getServerSideProps : GetServerSideProps = async (context) => {
     const queryClient = new QueryClient();
-    const { orgId } = context.params;
+    const { orgId } = context.params!;
```

Comment:
Apparently, when there are no parameters in a URL (which depends on the path and file name of this route handler) NEXT will not set `params` to `{}` (as one would expect) but instead not set it at all.

Because whether or not `params` exists depends on the path to this route, and that is decided in the source path (`src/pages/o/[orgId]/campaigns.tsx`), it is known at this point that `params` will never be null.

This seems like the perfect use case for non-null assertions (`!`). So the fact that eslint complains about them tells me that we should probably disable that rule globally and just be careful not to overuse the feature.

Reply:
Disable `@typescript-eslint/no-non-null-assertion` on these lines only.